src/charm.py

```diff
@@ -54,26 +54,48 @@ def __init__(self, *args):
         super().__init__(*args)
 
         self._name = "prometheus"
-        self._port = 9090
+        self._prometheus_port = 9090
+        self._prometheus_configurer_port = 9100
 
-        self.service_patch = KubernetesServicePatch(self, [(f"{self.app.name}", self._port)])
+        self.service_patch = KubernetesServicePatch(
+            self,
+            [
+                (f"{self.app.name}", self._prometheus_port),
+                (
+                    "prom-configmanager",
+                    self._prometheus_configurer_port,
+                    self._prometheus_configurer_port,
```

Comment:
Is this needed?

Reply:
It is - without it k8s service will not expose configurer's port.

Reply:
You're right. Removed. 
@ghislainbourgeois: That sounds like a nice idea. Wanna give it a shot?

Reply:
I would be happy to give it a shot, do you happen to know the upstream project for this lib? I looked in [canonical/observability-libs](https://github.com/canonical/observability-libs) but it seems to only have a `v1` and I am a bit confused.